### DISCUSSION TITLE
Documentation for WebDJ

### DISCUSSION THREAD

**Discussion by PDenzinger** [AzuraCast/AzuraCast#4368] [Q&A]
Is there any documentation for the WebDJ?

I don't know what the cue button does next to "Start/Stop Streaming" and the Microphone input.

Maybe an actual DJ would know what buttons like this do, but I do not.

**Comment by DrakeAedus** [comment]
"Cue" allows you to hear the output going to your mux.

If you have headphones or speakers assigned in the software, then you can listen live to the playlist or to your mic.
